### ISSUE TITLE
A11y: Content is not perceivable and/or accessible without requiring scrolling in two dimensions at 400% zoom

### ISSUE THREAD

**Issue by ChristofSimon** [grafana/grafana#82169]
**Steps to reproduce**:
start with viewport width of 1280 CSS pixels wide and zoom to 400%

**Actual Result**:
Parts of the content don’t reflow adequate.
1st example home/settings: it has to be scrolled horizontally to perceive all content
![1 4 10_reflow_settings](https://github.com/grafana/grafana/assets/122528177/a6051196-1ac8-4e84-b081-d6c1be771e2e)

2nd example diagram: scrollbar for diagram not accessible
![1 4 10_reflow_diagram](https://github.com/grafana/grafana/assets/122528177/af83b1c5-2dd1-4e11-8026-2be880ae8812)

3rd example "Edit Panel": In the "Edit Panel" mode, you can only see a small section of the page. Scrolling is not possible so that essential content is not accessible
![1 4 10_reflow_edit panel](https://github.com/grafana/grafana/assets/122528177/e294172b-36da-4b3b-97bf-7447bc57142d)

**Expected Result**
Content can be presented without loss of information or functionality, and without requiring scrolling in two dimensions.

**Relevant WCAG Criteria:** [#.#.# WCAG Criterion](link to https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0)
1.4.10 Reflow: https://www.w3.org/TR/WCAG21/#reflow

**Environment**:
- Grafana version: 9.1.0
- Data source type & version:
- User OS & Browser: Windows 10, Chrome Version 107.0.5304.88
- Others:

**Comment by tonypowa** [comment]
cc @grafana/grafana-frontend-platform
